--- include/nasmlib.h	2018-02-07 13:40:15.000000000 -0800
+++ include/nasmlib.h	2018-09-10 18:18:09.563556571 -0700
@@ -191,8 +191,8 @@
  * seg_init: Initialise the segment-number allocator.
  * seg_alloc: allocate a hitherto unused segment number.
  */
-void pure_func seg_init(void);
-int32_t pure_func seg_alloc(void);
+void seg_init(void);
+int32_t seg_alloc(void);
 
 /*
  * many output formats will be able to make use of this: a standard
